CognitiveSearch.Skills/host.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Why is this file needed?

Reply:
This is a file that gets created when you create an azure function project in VS. Based on the [docs](https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json), it holds various settings about the azure function. In many cases, it'll just specify that version 2 is used. Looks to me like it get's updated after publishing so it may make sense to keep the file around